benchmarks/src/main/java/com/yahoo/omid/benchmarks/tso/TSOServerBenchmarkConfig.java

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright 2011-2016 Yahoo Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yahoo.omid.benchmarks.tso;
+
+import com.google.inject.AbstractModule;
+import com.yahoo.omid.YAMLUtils;
+import com.yahoo.omid.benchmarks.utils.Distribution;
+import com.yahoo.omid.benchmarks.utils.TimeValueTimeUnit;
+import com.yahoo.omid.metrics.MetricsRegistry;
+import com.yahoo.omid.tools.hbase.SecureHBaseConfig;
+import com.yahoo.omid.tsoclient.OmidClientConfiguration;
+
+public class TSOServerBenchmarkConfig extends SecureHBaseConfig {
+
+    private static final String DEFAULT_CONFIG_FILE_NAME = "tso-server-benchmark-config.yml";
+    private static final String CONFIG_FILE_NAME = "default-tso-server-benchmark-config.yml";
+
+    private TimeValueTimeUnit runFor;
+
+    private int txRunners;
+    private int txRateInRequestPerSecond;
+    private TimeValueTimeUnit warmUp;
+    private Distribution requestDistribution;
```

Comment:
Consider injecting an instance of Generator directly -> then you would not need an enum and GeneratorUtils classes at all